docs: update troubleshooting in wireguard section (#405)
foundriesio/docs: source/reference-manual/remote-access/wireguard.rst

Wireguard uses UDP. This can be difficult to troubleshoot. A very common problem
is when the VPN server has a firewall blocking traffic to the Wireguard port.

.. note::

>   When configuring a server behind a firewall make sure the desired port

Author: Done

  is passed through to the host running the server.

When trying to activate the server you get::

  ERROR: A UDP socket is already opened on 165.227.222.126:5555

> After making sure no other service is using that port, add ``--no-check-ip``

Reviewer: ```suggestion
Make sure no other service is using that port, then add ``--no-check-ip``
```

"...making sure no other service is using the port" Should be given as an imperative/instruction. 

This is also a bit ambiguous... Do you mean that any service using the port needs to be closed first, and then run the command with the `--no-check-ip` flag? It is also unclear what command this is being added to, though I'm assuming wireguard given the context.

Author: I separated the instruction from the corrective action
